include-build/roborazzi-gradle-plugin/src/test/java/io/github/takahirom/roborazzi/VersionComparisonTest.kt

```diff
@@ -0,0 +1,57 @@
+package io.github.takahirom.roborazzi
+
+import org.junit.Test
+import org.junit.Assert.assertTrue
+import org.junit.Assert.assertFalse
+
+class VersionComparisonTest {
+
+    @Test
```

Comment:
mega minor:
I think a parameterized test makes more sense here, since Junit uses hard asserts by default: once the first assert fails, the test does not execute more code.

That is the reason why soft-assertions where created.
https://www.baeldung.com/java-assertj-soft-assertions

And with Parameterized tests, every assert would run in a separate test, which it is even better than soft-assertions when possible, in my opinion.
This should not be a blocker though, feel free to change it or not :)

Reply:
Ah, I see how it works now. It wraps and collects every error to create a single AssertionError. Thanks for clarifying.

Anyway, I've addressed your suggestion in this commit: 6a3c1a5